### DISCUSSION TITLE
How can I control interpolation?

### DISCUSSION THREAD

**Discussion by kimmolinna** [fonsp/Pluto.jl#1398] [Questions about using Pluto] [answered]
How do I prevent an interpolation for the second line?

```
@htl("""
	<p> Hello $(who)!</p>
	{prevent interpolation for this}("Hello $(who)!"){prevent interpolation for this}
""")
```
so the output would be like 
```
Hello 🌍!
Hello $(who)!
```

This would be useful with d3. For example [d3.hierachy ](https://observablehq.com/@d3/d3-hierarchy) uses `transform` and I don't figure out how to execute the following line
```
.attr("transform", `translate(${marginLeft},${dx - x0})`);
```

**Comment by fonsp** [comment, marked as answer]
You can use `\$` instead of `$` to write a dollar sign. Have fun!
